ui/components/appbar-primary.jsx

```diff
@@ -82,15 +83,18 @@ module.exports = function(core, config, store) {
 			window.open(stringUtils.stripQueryParam(window.location.href, "embed"), "_blank");
 		},
 
+		badgeFilter: function(note) {
+			return note.score >= 30;
+		},
+
 		showNotifications: function(event) {
```

Comment:
why are we using dom manipulation to add notification center? Why not directly add to appbar's render()?